src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,375 @@
+      "data_packs": "数据包",
+      "image_packs": "图像包"
+    },
+    "cores": "核心",
+    "edit": "编辑"
+  },
+  "platforms": {
+    "cancel": "取消",
+    "controls": {
+      "data_packs": "数据包",
+      "image_packs": "图像包",
+      "search": "搜索"
+    },
+    "data_packs": {
+      "apply_changes": "共执行了 {count, plural, =0 {0项修改} one {1项修改} other {# 项修改}}",
+      "close": "关闭",
+      "current": "现有"
+    },
+    "delete_unused": "删除 {count, plural, one {# 个未被使用的平台} other {# 个未被使用的平台}}?",
+    "edit": "编辑",
+    "image_packs": {
+      "apply_changes": "共执行了 {count, plural, =0 {0项修改} one {1项修改} other {# 项修改}}",
+      "close": "关闭",
+      "current": "现有"
+    },
+    "save": "保存"
+  },
+  "progress": {
+    "remaining_time": "剩余时间 (approx): {remainingTime}"
+  },
+  "save_info": {
+    "confirm_restore": "确认使用备份恢复?",
+    "controls": {
+      "back": "返回列表",
+      "hide_unchanged": "隐藏未变更项目",
+      "search": "搜索"
+    },
+    "current": "目前",
+    "date": "{date, date, short}",
+    "newer": "更新",
+    "older": "更旧",
+    "time": "{date, time, short}"
+  },
+  "save_states": {
+    "cartridge": "卡带",
+    "confirm_delete": "您确认删除 {count, plural, =0 {0个即时存档} one {1个即时存档} other {# 个即时存档}}?",
+    "controls": {
+      "clear_selection": "取消勾选",
+      "delete_selection": "删除勾选 ({count})",
+      "search": "搜索"
+    },
+    "count": "( {count} / 128 )",
+    "item": {
+      "date": "{date, date, long}",
+      "time": "{date, time, short}"
+    },
+    "photos": {
+      "title": "图片输出",
+      "palette": "调色板",
+      "custom": "自定义",
+      "close": "关闭",
+      "export_selected": "输出选择项",
+      "export_all": "输出所有",
+      "colours": {
+        "gb": "GB",
+        "gb_player": "GB Player",
+        "black_and_white": "黑&白"
+      }
+    }
+  },
+  "saves": {
+    "backups": "备份: {count}",
+    "confirm_delete": "您确认需要移除这个备份目录?",
+    "controls": {
+      "add_backup_location": "添加备份目录",
+      "mister_sync": "MiSTer Sync"
+    },
+    "latest_backup": "最后的备份: {date, date, long}, {date, time, short}",
+    "none": "添加以上备份目录",
+    "not_found": "未找到",
+    "oldest_backup": "最早的备份: {date, date, long}, {date, time, short}",
+    "remove": "移除目录",
+    "view_saves": "浏览存档"
+  },
+  "screenshot_info": {
+    "controls": {
+      "back": "返回列表",
+      "save": "保存",
+      "share": "分享",
+      "upscaled": "画质提升"
+    }
+  },
+  "screenshots": {
+    "controls": {
+      "delete_selected": "删除选择 ({count})",
+      "export_selected": "输出选择 ({count})",
+      "search": "搜索",
+      "select": "选择"
+    },
+    "no_screenshots": "您的截屏都会在此显示"
+  },
+  "settings": {
+    "3d_pocket": {
+      "title": "Pocket颜色:",
+      "black": "黑",
+      "white": "白",
+      "glow": "荧光"
+    },
+    "alternate_skip": {
+      "checkbox": "跳过可选资源",
+      "ramble_1": "This setting will skip processing json files under the `_alternatives` folder,\nmeaning you'll just get the \"main\" titles for the cores which sort their JSON files this way.", //not fully understand
```

Comment:
When it's searching for files to install there's some which are listed in a folder named `_alternatives`, most users don't care about them so it skips what's in there if this setting is true

Reply:
ok